extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala

```diff
@@ -246,10 +246,10 @@ class HudiDataSourceV2RelationTableExtractor extends TableExtractor {
     invokeAs[LogicalPlan](v1, "table") match {
       // Match multipartIdentifier with tableAlias
       case SubqueryAlias(_, SubqueryAlias(identifier, _)) =>
-        new StringTableExtractor().apply(spark, identifier.toString())
+        lookupExtractor[StringTableExtractor].apply(spark, identifier.toString())
```

Comment:
any special reason for changing this? `new` seems straightforward

Reply:
No much. Just a minor improvement to reuse the stateless extractor instance. `lookupExtractor` is used all other 29 places.

Reply:
What's the state for stateless here? lookupExtractor seems much heavier for such a case

Reply:
The extractor does not own a state. And they are more like snippets that cooperate together for object extractions, so we are reusing them rather than creating an instance inside another. No much heavier in code lines and has less memory footprint.